smart_campus/app/templates/app/station_edit.html

```diff
@@ -3,129 +3,119 @@
 
 {% block stylesheet %}
   {{ block.super }}
-  <link rel="stylesheet" type="text/css" href="{% static "app/css/station_edit.css" %}" />
+  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.css">
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.js"></script>
+	<link rel="stylesheet" type="text/css" href="{% static "app/css/edit.css" %}">
```

Comment:
Indentation.

Reply:
okay, indented it.

Reply:
Well...
What I mean is that your indentation is not correct, not simply add indentations.